bundles/mapping/mapmodule/plugin/getinfo/request/SwipeStatusRequest.js

```diff
@@ -0,0 +1,59 @@
+/**
+ * @class Oskari.mapframework.mapmodule.getinfoplugin.request.SwipeStatusRequest
+ * Saves swipe tool status to GetInfoPlugin.
+ *
+ * Requests are build and sent through Oskari.Sandbox.
+ * Oskari.mapframework.request.Request superclass documents how to send one.
+    */
+Oskari.clazz
+    .define('Oskari.mapframework.mapmodule.getinfoplugin.request.SwipeStatusRequest',
+        /**
+         * @method create called automatically on construction
+         * @static
+         *
+         * @param {Number} layerId layer ID
+         * @param {Boolean} active swipe active
+         * @param {Number} cropX swipe tool X position
+         */
+        function (layerId, active, cropX) {
+            this._layerId = layerId;
+            this._active = active;
```

Comment:
I don't think we need the boolean here. If a layerId is present we can deduce that it's active?